tests.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $1 ]; then
+    echo "Stack name not passed, exiting"
+    exit 1
+fi
+
+STACK=$1
+echo "$STACK"
+
+
+function clean {
+
+    ## Cleanup DB
+
+
+    for f in "${TEST_FILES[@]}"; do
+
+        FILE_NAME=$(cut -d "/" -f2- <<< "$f")
+
+        echo "Removing item from DB : $FILE_NAME"
+
+        aws dynamodb delete-item --table-name $DYNAMO_TABLE \
+        --key "{\"id\": {\"S\": \"s3://$BUCKET_IN/$FILE_NAME\"}}"
+
+
+        if [ $? -ne 0 ]; then
+            echo -e "Remove item failed" ; exit 1
+        fi
+
+    done
+
+    ## Cleanup Output
+
+
+    for f in "${TEST_FILES[@]}"; do
+
+      FILE_NAME=$(cut -d "/" -f2- <<< "$f")
+
+      echo "Removing from output : ${FILE_NAME/md/html}"
+
+      aws s3 rm s3://"$BUCKET_OUT"/"${FILE_NAME/md/html}"
```

Comment:
It's unclear to me what this line does.  I'm not familiar with the syntax `"${FILE_NAME/md/html}`.

Reply:
Added comment with explanation. This is in order to substitute the .md extension for .html as the files expected in the output folder should be the same as those from the test folder but with a html extension.